### PR TITLE
Add Checkmark On visited Links

### PR DESCRIPTION
**Pull request by Xipas**
Added the missing code to show the checkmark on visited links on sidebar. All js code was already in place, just missing some html.

### CODE REVIEW

**Comment by matcornic** [comment]
Many thanks @Xipas for your PR :)

I did not implemented this functionality because, personally, I did not find it useful. But other people (like you 😋) do.

So, can you add an optional boolean `showVisitedLinks` parameter [like this](https://matcornic.github.io/hugo-learn-doc/basics/configuration/#site-configuration:d680e8a854a7cbad6d490c445cba2eba) ? Then use `.Site.Params.showVisitedLinks` in the Go Template.

Thank you for your understanding.

**Comment by Xipas** [comment]
Done :)

**Comment by Xipas** [comment]
Added a css code to set color and align to right.

**Comment by matcornic** [comment]
Updated the doc as well : https://matcornic.github.io/hugo-learn-doc/basics/configuration/

File: static/css/theme.css [hunk 1]
```diff
@@ -988,4 +988,21 @@ pre .copy-to-clipboard:hover {
     transform-style: preserve-3d;
 }
 
+/* Add checkmark icon align to right and color*/
+#sidebar ul.topics > li > a .fa {
+    margin-top: 9px;
+}
+#sidebar ul li .fa {
```
Comment (matcornic) [inline static/css/theme.css]: @Xipas Did not see it at the first time, but if we define an font awesome icon in the prefix of a menu item, it's not shown. We should add a class for the check icon, in order to be specific.

**Comment by matcornic** [comment]
@Xipas I added a comment on the code of your PR. I got a regression when I used Font Awesome icons as prefix for the menu (ex in front matter of a chapter page : `icon: "<i class='fa fa-users'></i>"`)

**Comment by matcornic** [comment]
@Xipas it's fixed in de8584e890e444b3652e6dca8f6f2b167f494aee

**Pull request merged**